etc/aysnc_events.xml

```diff
@@ -7,11 +7,20 @@
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Aligent_AsyncEvents:etc/async_events.xsd">
```

Comment:
looks like the name of this file has a typo. should be changed from `aysnc_events.xml` to  `async_events.xml`